src/gsy_e/setup/tekniker_cop_model/README.md

```diff
@@ -0,0 +1,67 @@
+# Individual COP model integration
+
+This document describes to train COP models, developed by [Tekniker](https://www.tekniker.es/en/) by deploying the [DOE-2](https://www.doe2.com/) building energy analysis program, to yield a statistical model by leveraging measurements of select heat pump manufacturers under predefined operational conditions.
+
+## COP model training
+
+The goal of training the COP model is to produce a JSON file that contains the COP model parameters for a specific heat pump.
+For training the COP model, performance data for the selected heat pump is needed, that comply to the characteristics listed in the following section.
```

Comment:
```suggestion
The training of the COP model produces a JSON file that contains the COP model parameters for a specific heat pump. This will later be used by the HeatPumpStrategy model, in order to more accurately approximate the select heat pump model.
For training the COP model, performance data for the selected heat pump is needed, that comply to the characteristics listed in the following section.
```